src/appleseed.studio/mainwindow/rendering/renderwidget.cpp

```diff
@@ -231,7 +233,16 @@ void RenderWidget::highlight_tile(
 
     // Draw a bracket around the tile.
     const size_t BracketExtent = 5;
-    const std::uint8_t BracketColor[3] = { 255, 255, 255 };
+    std::uint8_t BracketColor[3] = { 255, 255, 255 }; // white
```

Comment:
Same remarks here.

In the meantime I'll try to refactor this code to share the `RenderWidget` class across appleseed.studio and appleseed.bench.